receiver/postgresqlreceiver/integration_test.go

```diff
@@ -26,6 +28,14 @@ func TestIntegration(t *testing.T) {
 	t.Run("single_db", integrationTest("single_db", []string{"otel"}))
 	t.Run("multi_db", integrationTest("multi_db", []string{"otel", "otel2"}))
 	t.Run("all_db", integrationTest("all_db", []string{}))
+
+	// Simulate enable separateSchemaAttrGate
+	err := featuregate.GlobalRegistry().Set(separateSchemaAttrID, true)
+	require.NoError(t, err)
```

Comment:
nit: Can we use [`testutil.SetFeatureGateForTest`](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/1b6a4517b5323e7cbd9ad97e1c7e5a8e2a9054e4/internal/common/testutil/testutil.go#L129)?

Reply:
ah thanks, I copied this from the mongodb tests but that would be cleaner I think, I'll adjust that